heron/cli/src/python/cli_helper.py

```diff
@@ -72,7 +72,7 @@ def run(command, parser, cl_args, unknown_args, action):
         "--command", command,
     ]
 
-    if opts.verbose():
+    if Log.getEffectiveLevel() == logging.DEBUG:
```

Comment:
this does not seem to be correct. if the --verbose option is provided by the user then only we add the --verbose option to the java programs.